plugins/dbviewer/frontend/public/templates/dbviewer.html

```diff
@@ -138,7 +168,7 @@ <h3>{{value.name}}</h3>
 							<table class="jh-type-object jh-root" id="dbviewer-collections">
 							{{#eachOfArray data.collections}}
 							<tr>
-								<th class="jh-key jh-object-key"><p><strong>_id:</strong> {{value._id}} <span class="jh-type-string"><a href='#/manage/db/{{../db}}/{{../collection}}/{{value._id}}' >View</a></span></p></th>
+								<th class="jh-key jh-object-key"><p><strong>_id:</strong> {{value._id}} <span id="dbviewer-view-link" class="jh-type-string"><a href='#/manage/db/{{../db}}/{{../collection}}/{{value._id}}' >View</a></span></p></th>
```

Comment:
Maybe id="dbviewer-view-link" should be class rather than id, as it will be multiple HTML elements with same id, no?